### PR TITLE
Long Texts

### PR DESCRIPTION
**Pull request by YashBit**
Solved the problem of long texts in Scenes :
1.Radar
2.Tactics of Sexual Predators
3.Bystander Intervention
4.Saftey Plan Basics
5.Safety Plan
6.Confidentiality
7.Support Services
8.Available Services after sexual assault
9. Peace Corps Commitment

### CODE REVIEW

**Comment by YashBit** [comment]
@chamika  I have made the changes .

File: PCSA/ScrollableTextView.swift [hunk 1]
```diff
@@ -0,0 +1,32 @@
+//
```
Comment (chamika) [inline PCSA/ScrollableTextView.swift]: There are two ScrollableTextView.swift files. Need to remove one. I think this file is not in the correct location

Comment (chamika) [inline PCSA/ScrollableTextView.swift]: Following issues described in https://github.com/systers/FirstAide-iOS/pull/32#issuecomment-266463555
is still there 
Safety Plan Worksheets -> Slide 3 Actions pop up
Support Services -> Available Services After .. -> Learn More About reporting types -> Slide 1

Comment (YashBit) [inline PCSA/ScrollableTextView.swift]: I haven't made any changes to the Safety Plans worksheets scene . I don't know why the action label would pop up .

**Pull request closed without merging**